### DISCUSSION TITLE
Fix permissions for `phpfpm-apache`

### DISCUSSION THREAD

**Discussion by o1y** [shinsenter/php#23] [General]
Hi! This seems to be a really cool project :)

Unfortunately, I don't understand how to set correct permissions when using `phpfpm-apache`. I'm always getting Permission errors, like `Failed to open stream: Permission denied` for temporary files created by my php application. In my Docker Compose `service` I'm setting correct values for `PUID` and `PGID`. Do I have to manually fix permissions using an init script or so?

Directories are being created as `root:root` and not `webuser:webgroup`.

Thanks!

**Comment by shinsenter** [comment]
@o1y 

The exact solution will depend on your specific setup and configuration.
If you are using Docker on Windows Subsystem for Linux (WSL), there are a few things in #22 to consider when setting up file permissions.
